vertical-pod-autoscaler/hack/run-e2e.sh

```diff
@@ -20,6 +20,42 @@ set -o pipefail
 
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 
-${SCRIPT_ROOT}/hack/deploy-for-e2e.sh $*
-go test ${SCRIPT_ROOT}/e2e/*go -v  --args --ginkgo.v=true --ginkgo.focus="\[VPA\]"
+function print_help {
+  echo "ERROR! Usage: run-e2e.sh <suite>"
+  echo "<suite> should be one of:"
+  echo " - recommender"
+  echo " - updater"
+  echo " - admission-controller"
+  echo " - full-vpa"
+}
+
+if [ $# -eq 0 ]; then
+  print_help
+  exit 1
+fi
+
+if [ $# -gt 1 ]; then
+  print_help
+  exit 1
+fi
+
+SUITE=$1
+
+case ${SUITE} in
+  recommender|updater|admission-controller|full-vpa)
+    ${SCRIPT_ROOT}/hack/vpa-down.sh
+    ${SCRIPT_ROOT}/hack/deploy-for-e2e.sh ${SUITE}
+
+    # VPA creation and listing for debugging test-infra. DELETE AFTER DEBUG
```

Comment:
Is this meant to get into the repo or is it a leftover from the test-infra issues?

Reply:
It is meant to get into the repo. I am trying to debug https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-autoscaling-vpa-recommender/73 which I cannot reproduce locally.